CONTRIBUTING.md

```diff
@@ -0,0 +1,586 @@
+
+## Submitting issues around vector tile quality
+
+It's important to get feedback about the quality of local tile results. Your local knowledge will make it easier for us to understand the problem. When submitting issues be sure to include:
+
+- Where in the world you were looking (a placename &/or zoom, latitude, longitude)
+- Your tile query
+- Your expected result
+- Your actual result
+
+
+# Pull Requests Welcome!
+
+## Project standards overview
+
+Tilezen has several miscellaneous standards:
+
+- we follow [PEP8](https://www.python.org/dev/peps/pep-0008/) coding style for Python and use [Flake8](http://flake8.pycqa.org/en/latest/) to enforce those conventions
+- we *love* tests, [check them out](https://github.com/tilezen/vector-datasource/tree/master/integration-test)
+- we use [CircleCI](https://circleci.com/gh/mapzen/vector-datasource) for continuous integration testing
+- we use [semver](http://semver.org/) for package versioning
+
+All  unit tests in a project will be automatically invoked when you commit to an existing project; make sure they exit successfully!
+
+## Active contributors
+
+We'll gladly invite active contributors to become members of the [Tilezen organization](https://github.com/Tilezen). New members will gain direct write permissions, *and with great power comes great responsibility*.
+
+## Project overview (what to change, where)
+
+Generally speaking there are three aspects of developing vector tiles.
+
+- Configuring project setup, see [wiki page](https://github.com/tilezen/vector-datasource/wiki/Mapzen-Vector-Tile-Service)
+- Updating database properties (can be done ahead of time or at runtime)
+- Changing how features are selected from the database (requires TileServer restart)
+
+<div class='alert-message'>
+Yellow call-outs like this are meant to draw your attention to an important idea or distinction you should keep in mind.
+</div>
+
+Map database in Postgres stores data from OpenStreetMap and other projects like Natural Earth and Who's On First.
+
+When data is loaded, database "triggers" calculate if a feature is included in which layer(s), at what "minimum zoom", and other Mapzen specific "mz" properties.
+
+When modifying the logic below, we'll need to update our Postgres functions, migrate the data, and cut new tiles.
+
+### Changing tile content in the vector-datasource repo
+
+Most content changes (eg: adding a new kind of feature) only require a database modification. Content changes are configured using YAML files which specify which database features are "filtered" and outputed in tiles. The location for these content filters is in the `yaml/` directory, for example: [pois.yaml](yaml/pois.yaml).
+
+Some preexisting feature filters are configured using an older raw SQL format in `jinja` files in the `queries/` directory. This older syntax is still helpful to select multiple feature properties across many kinds of features at once. For example, [pois.jinja](queries/pois.jinja).
```

Comment:
Spell "pre-existing".

We might want to take a moment first to describe the pipeline in general and how everything fits together more. The yaml files help manage filtering and typical property value rules, especially the feature's `kind`. These get folded into the sql, which is generated via jinja templates. And then we have python processing that occurs per feature and across layers, which is useful for more involved logic.

I wouldn't quite frame it as jinja is old and yaml is new, but more that yaml is for the typical differences between features in a layer, and the jinja is better suited for rules that apply to all features in a layer.
But generally speaking, I'd expect most typical contributions or updates to occur at the yaml level.